spec.html

```diff
@@ -64,7 +61,7 @@
 <div id="metadata-block">
   <h1>About this Specification</h1>
   <p>The document at <a href="https://tc39.es/ecma262/">https://tc39.es/ecma262/</a> is the most accurate and up-to-date ECMAScript specification. It contains the content of the most recent yearly snapshot plus any <a href="https://github.com/tc39/proposals/blob/HEAD/finished-proposals.md">finished proposals</a> (those that have reached Stage&nbsp;4 in the <a href="https://tc39.es/process-document/">proposal process</a> and thus are implemented in several implementations and will be in the next practical revision) since that snapshot was taken.</p>
-  <p>This document is available as <a href="">a single page</a> and as <a href="multipage/">multiple pages</a>.</p>
+  <p>This document is available as <a href>a single page</a> and as <a href="multipage/">multiple pages</a>.</p>
```

Comment:
i don't think these are the same - isn't `<a href>` equivalent to `<a href="href">`?

Reply:
> isn't `<a href>` equivalent to `<a href="href">`?

No.